doc/ref/helpintf.xml

```diff
@@ -90,25 +90,20 @@ any more. The book can be re-installed using <Ref Func="HELP_ADD_BOOK" />. <P/>
 <Section Label="The manual.six File">
 <Heading>The manual.six File</Heading>
 
-If a <F>manual.six</F> file for a help book is not in the format of the
-<F>gapmacro.tex</F> macros, explained in the document 
-<Q>The gapmacro.tex Manual Format</Q> (see the file 
-<F>gap4r5/doc/gapmacrodoc.pdf</F> from the tools archive 
```

Comment:
So, do you prefer to throw away mentioning of `doc/gapmacrodoc.pdf` completely? I thought we build it and include in the GAP source distribution.